jaggr-core/src/main/java/com/ibm/jaggr/core/cache/IGenericCache.java

```diff
@@ -81,4 +71,12 @@
 	 * @param aggregator The aggregator instance for the cache
 	 */
 	public void setAggregator(IAggregator aggregator);
+
+	/**
+	 * Returns a new, empty, instance of this cache object. Used by the cache manager when clearing
+	 * the aggregator cache.
+	 *
+	 * @return a new instance of this cache object
+	 */
+	public IGenericCache newInstance();

```

Comment:
I worry that the listener and this change (the way they are documented) are going to make it harder to integrate a generic caching framework like ehcache in the future.

Can you explain what was happening that we needed this?